oper8/reconcile.py

```diff
@@ -79,14 +78,16 @@ class ReconciliationResult:
     exception: Exception = None
 
 
+# Forward declarations of Controller
+CONTROLLER_TYPE = "Controller"
```

Comment:
Should the forward declarations all be in the same place? Maybe in `consts`?

Reply:
I thought about that. I was trying to follow the `c++` paradigm of putting them in the "header" section of the individual file to handle circular dependencies.

Reply:
I think that makes sense, it duplicates some code but will be a bit clearer in the long run